CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## main / unreleased
 
+* [FEATURE] Querier: add experimental streaming PromQL engine, enabled with `-querier.query-engine=mimir`. #9639
+
```

Comment:
This is in the wrong spot, it should be a few lines down, under "Grafana Mimir".